scripts/us_census/pep/us_pep_sexrace/postprocess.py

```diff
@@ -0,0 +1,370 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Script to read all
+the generated csv for
+National, State and County
+and generate final csv, MCF, TMCF file
+"""
+
+import pandas as pd
+import os
+
+_CODEDIR = os.path.dirname(os.path.realpath(__file__))
+
+
+def create_single_csv(output_files_names):
+    """
+    Function generate 3 csv
+    1. preprocess.csv : for the files which are processed as is.
+    2. preprocess_aggregate.csv : for the files which
+        are having Count Person Male/Female aggregated.
+    3. preprocess_aggregate_state_2010_2020.csv : for the files
+        which are aggregated from different geo granularity.
+
+    Args:
+        output_files_names: nested list of output file names generated.
+    """
+    # list of output files which are having no aggregation
+    as_is_output_files = output_files_names[1]
+
+    # list of output files which are having aggregation Count_Person_Male
+    # and Count_Person_Female
+    aggregate_output_files_before = output_files_names[2]
+
+    # list of output files which are having aggregation Count_Person_Male
+    # and Count_Person_Female
+    aggregate_output_files_after = output_files_names[3]
+
+    # aggregated State values 2010-2020 from County 2010-2020 data
+    geo_aggregate_output_files = output_files_names[4]
+
+    df1 = pd.DataFrame()
+    df3 = pd.DataFrame()
+    df5 = pd.DataFrame()
+    df7 = pd.DataFrame()
+
+    column_names = {}
+    # aggregating the files which are processed as is
+    # to final output csv
+    if len(as_is_output_files) > 0:
```

Comment:
nit: create_single_csv() is pretty long, maybe each of the "if" blocks can be abstracted out as a function?

Reply:
Reduces the length of create_single_csv(). Abstracted content of 'if' blocks out as function.